src/hyperlight_host/src/hypervisor/mod.rs

```diff
@@ -208,12 +208,19 @@ pub(crate) trait InterruptHandleImpl: InterruptHandle {
 pub trait InterruptHandle: Send + Sync + Debug {
     /// Interrupt the corresponding sandbox from running.
     ///
-    /// - If this is called while the the sandbox currently executing a guest function call, it will interrupt the sandbox and return `true`.
-    /// - If this is called while the sandbox is not running (for example before or after calling a guest function), it will do nothing and return `false`.
+    /// This method sets a cancellation flag that prevents or stops the execution of guest code.
+    /// The effectiveness of this call depends on timing relative to the guest function call lifecycle:
+    ///
+    /// - **Before guest call starts** (before `clear_cancel()` in `MultiUseSandbox::call()`):
+    ///   The cancellation request will be cleared and ignored.
+    /// - **After guest call starts but before entering guest code** (after `clear_cancel()`, before `run_vcpu()`):
+    ///   Will prevent the guest from executing.
+    /// - **While executing guest code**: Will interrupt the vCPU.
+    /// - **After guest call completes**: Has no effect (cancellation is cleared at the start of the next call).
```

Comment:
this comment is more confusing that before. keep the same comment just remove references to true and false